ogre2/src/media/materials/scripts/thermal.material

```diff
@@ -98,3 +98,49 @@ material ThermalHeatSource
     }
   }
 }
+
+// copied from gaussian_noise.material
+// TODO(adlarkin) try to reference the GaussianNoiseVS definition in
+// gaussian_noise.material so that we don't have a duplicate definition here?
+vertex_program GaussianNoiseVS glsl
+{
+  source gaussian_noise_vs.glsl
+  default_params
+  {
+    param_named_auto worldViewProj worldviewproj_matrix
+  }
+}
```

Comment:
I think they have to be in the same `.material` file for it to work. Can you rename this to `HeatSignatureVS`? Just to be safe so that when the gaussian noise material gets loaded, there won't be conflicts in the  vertex program references.

Reply:
renamed in https://github.com/ignitionrobotics/ign-rendering/pull/189/commits/f710f72c018e0caf03a6a77a29881302d4b9263f